onchange_helper/models/base.py

```diff
@@ -31,7 +31,12 @@ def play_onchanges(self, values, onchange_fields):
         # If self is an empty record we will have an empty value
         if self:
             self.ensure_one()
-            record_values = self._convert_to_write(self.read()[0])
+            record_values = self._convert_to_write(
```

Comment:
@lmignon , What do you think about this?

Reply:
@sbejaoui  LGTM and will avoid 1 useless SQL query on each call to the method